packages/alfa-rules/src/sia-r113/rule.ts

```diff
@@ -32,13 +33,31 @@ export default Rule.Atomic.of<Page, Element>({
         // Existence of a bounding box is guaranteed by applicability
         const box = target.getBoundingBox(device).getUnsafe();
         const name = WithName.getName(target, device).getOr("");
+
+        const tooCloseNeighbors: Array<Element> = [];
         return {
           1: expectation(
             isUserAgentControlled()(target),
             () => Outcomes.IsUserAgentControlled(name, box),
-            hasSufficientSizeOrSpacing(document, device)(target)
-              ? () => Outcomes.HasSufficientSizeOrSpacing(name, box)
-              : () => Outcomes.HasInsufficientSizeAndSpacing(name, box),
+            () =>
+              expectation(
+                hasSufficientSize(24, device)(target),
+                () => Outcomes.HasSufficientSize(name, box),
+                () =>
+                  expectation(
+                    hasSufficientSpacing(document, device)(
+                      target,
+                      tooCloseNeighbors,
+                    ),
```

Comment:
**Suggestion:** We can replace `hasSufficientSpacing` by something that produce a sequence or other kind of iterable (typically `yield`ing them upon discovery) and store that value locally.

That function can be called **before** the call to `expectation` in the arrow function:
```typescript
expectation(hasSufficientSize,
() => …
() => { const tooCloseNeighbours = … ; return expectation(tooCloseNeighbours.isEmpty(), …, …) }
```